frontend/src/app/main/ui/auth/login.cljs

```diff
@@ -147,6 +149,11 @@
 
     [:& login-form {:params params}]
 
+    [:span.separator "OR"]
```

Comment:
Need to translate this

Reply:
And this text should not appear if oauth login is disabled (so there are no login buttons)